### PR TITLE
Add volta in package.json

### PR DESCRIPTION
**Pull request by NullVoxPopuli**
to help folks using volta use the correct versions of the specified tools

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -132,6 +132,10 @@
   "engines": {
     "node": "18.* || 20.* || >= 21"
   },
+  "volta": {
+    "node": "18.19.0",
```
Comment (bmish) [inline package.json:136]: Can we use Node 21?

Comment (NullVoxPopuli) [reply package.json:136]: I suppose, but convention has been to keep local environment on min-supported so you don't accidentally use a feature that's not in your min-supported node.

(exception being libraries that don't need node at all, such as browser-only libraries)

Comment (bmish) [reply package.json:136]: We have CI tests for all of our supported Node versions. I would prefer to rely on those to catch issues.

Comment (NullVoxPopuli) [reply package.json]: you got it :tada:

**Review by bmish — APPROVED** [review]
Thanks!

**Pull request merged**